Python/codegen.c

```diff
@@ -1243,10 +1243,10 @@ codegen_function_body(compiler *c, stmt_ty s, int is_async, Py_ssize_t funcflags
             _PyCompile_ExitScope(c);
             return ERROR;
         }
+        Py_ssize_t idx = _PyCompile_AddConst(c, docstring);
+        Py_XDECREF(docstring);
```

Comment:
I think now this can be 

```suggestion
        Py_DECREF(docstring);
```